src/components/Pricing/PricingCalculator/Tabbed.tsx

```diff
@@ -0,0 +1,452 @@
+                                People tab
+                            </Link>
+                        </li>
+                        <li>Filter on user properties</li>
+                        <li>
+                            Merge identities across <code className="text-sm">distinct_id</code> values (i.e. merging an
+                            anonymous user with a logged in user)
+                        </li>
+                        <li>Cohort creation</li>
+                        <li>Feature flag, experimentation, and survey targeting on user properties</li>
+                        <li>Tracking initial UTM values and referrers across anonymous and identified users</li>
+                    </ul>
+
+                    <p className="text-[15px] mb-0">
+                        Person profiles are split out from standard event prices so you don't have to pay for them if
+                        you don't need them.
+                    </p>
+
+                    <div className="mb-8">
+                        <CallToAction href="/docs/data/persons" size="sm" className="mt-4" external>
+                            Read the docs
+                        </CallToAction>
+                    </div>
+
+                    <h3 className="text-xl border-b border-light dark:border-dark pb-1 mb-2">
+                        Do you need person profiles?
+                    </h3>
+
+                    <div className="space-y-2 mb-8">
+                        <details>
+                            <summary className="cursor-pointer">
+                                <div className="inline-flex items-center justify-between w-[95%]">
+                                    <strong>"I want to track ALL my users"</strong>
+                                    <span>
+                                        <div className="border border-green rounded px-1 py-0.5 text-green font-bold uppercase text-sm">
+                                            Yes
+                                        </div>
+                                    </span>
+                                </div>
+                                <p className="text-sm opacity-75 pl-4 mb-0">
+                                    Anonymous website traffic <em>AND</em> signed in users
+                                </p>
+                            </summary>
+                            <div className="ml-4 my-2 border border-light dark:border-dark bg-accent dark:bg-accent-dark p-2 rounded [&_p]:text-sm">
+                                <p className="mb-0">
+                                    <strong className="text-green">Yes, you should enable this.</strong> It will allow
+                                    you to track visitors from the first time they visit your site and merge them with
+                                    an identified user record once they become a customer.
+                                </p>
+                            </div>
```

Comment:
I'd put this one last

```suggestion
                            <summary className="cursor-pointer">
                                <div className="inline-flex items-center justify-between w-[95%]">
                                    <strong>"I want to store details about some anonymous users"</strong>
                                    <span>
                                        <div className="border border-green rounded px-1 py-0.5 text-green font-bold uppercase text-sm">
                                            Yes
                                        </div>
                                    </span>
                                </div>
                                <p className="text-sm opacity-75 pl-4 mb-0">
                                    Some anonymous website traffic and all signed in users
                                </p>
                            </summary>
                            <div className="ml-4 my-2 border border-light dark:border-dark bg-accent dark:bg-accent-dark p-2 rounded [&_p]:text-sm">
                                <p className="mb-0">
                                    <strong className="text-green">Yes, you'll use person profiles any time you provide person information to store.</strong> This will allow you to store things like site preferences (eg. dark mode) and more on non-identified users and still leverage all the benefits of person profiles. 
                                </p>
                            </div>
```